--- a/drivers/net/ethernet/airoha/airoha_ppe.c
+++ b/drivers/net/ethernet/airoha/airoha_ppe.c
@@ -184,7 +184,7 @@ void airoha_ppe_set_mtu(struct airoha_g
 	ppe_id = !airoha_is_lan_gdm_dev(dev) && airoha_ppe_is_enabled(eth, 1);
 	index = port->id == AIROHA_GDM4_IDX ? 7 : port->id;
 
-	dev_info(eth->dev, "Setting PPE %d MTU for index %d to %u\n", ppe_id, index, mtu);
+	dev_info(eth->dev, "Setting PPE %d MTU for index %d to %u\n", ppe_id, index, len);
 	airoha_fe_rmw(eth, REG_PPE_MTU(ppe_id, index),
 		      FP_EGRESS_MTU_MASK(index),
 		      __field_prep(FP_EGRESS_MTU_MASK(index), len));
